src/workspace/src/hydroball_data_websocket/include/hydroball_data_websocket/hydroball_data_websocket.h

```diff
@@ -54,9 +54,10 @@ class TelemetryServer {
 
     void on_message(connection_hdl hdl, server::message_ptr msg) {
 	rapidjson::Document document;
-
         if(document.Parse(msg->get_payload().c_str()).HasParseError()){
         	//Not valid JSON
+        	ROS_ERROR("invalid json");
+        	std::cerr<<msg->get_payload().c_str()<<"\n";
```

Comment:
You can put line 59/60 in the ROS_ERROR call. ROS_ERROR is a C-style printf() wrapper. You can use printf flags to format error messages into it. Ex:

ROS_ERROR("Apples, oranges and %s", "bananas");

Or use ROS_ERROR_STREAM for a c++ stream syntax:

ROS_ERROR_STREAM("Apples, oranges and " << "bananas");